app/api/dao/user.py

```diff
@@ -50,6 +51,12 @@ def create_user(data):
         if 'available_to_mentor' in data:
             user.available_to_mentor = data['available_to_mentor']
 
+        try:
+            gravatar = Gravatar(email)
+            user.photo_url = gravatar.get_image(size=512)
+        except:
+            return messages.ERROR_WHILE_CALLING_GRAVATAR, 500
+
```

Comment:
We do not need this happening while user creation! Pulling a GRAVATAR during user creation (without verification) can have bad consequences. This entire feature should work only after the user is verified!